firebase-installations/src/main/java/com/google/firebase/installations/remote/FirebaseInstallationServiceClient.java

```diff
@@ -161,6 +164,8 @@ public InstallationResponse createFirebaseInstallation(
           return readCreateResponse(httpURLConnection);
         }
 
+        Log.w(FIS_TAG, readErrorResponse(httpURLConnection));
```

Comment:
How about creating a method for this to prevent code duplication:

```
private void logFisCommunicationError(HttpURLConnection conn) {
  String logString = [...];
  Log.w(FIS_TAG, logString);
}
```

Please ignore/ack if you think this makes no sense.

Reply:
Done